numpy/core/src/multiarray/descriptor.c

```diff
@@ -451,18 +446,23 @@ _convert_from_array_descr(PyObject *obj, int align)
         }
         else if (PyTuple_Check(name)) {
             if (PyTuple_GET_SIZE(name) != 2) {
-                _report_generic_error();
+                PyErr_Format(PyExc_TypeError, 
+				"If a tuple, the first element of a field tuple must have \
+				two elements, not %d",
```

Comment:
Thanks! Sorry for not pointing it out, but this %d is also incorrect. You cannot break the strings like this (if you check the initial whitespace is not gone). The solution is to use string concatenation if you wish to break the line:
```
PyErr_Format(Type,
        "This is a very                                                 long string "
        "which is continued on the next line."
```
One thing to note is the trailing space on the previous line. Also above, I like exactly 8 spaces indentation. You can also align with the `(`, but 8 spaces just gives you much more space to write the message.

Reply:
Good point, thanks for noticing that about the spacing. I just made a new commit updating it. Does that look better?

Reply:
Yeah, thanks, looks nice. You forgot one though :).

Reply:
Found it thanks. I think the rest of the backslash areas are code I didn't modify.